caravel/assets/spec/javascripts/sqllab/DatabaseSelect_spec.jsx

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import Select from 'react-select';
+import { DatabaseSelect } from '../../../javascripts/SqlLab/components/DatabaseSelect';
+import { shallow } from 'enzyme';
+import { describe, it } from 'mocha';
+import { expect } from 'chai';
+import sinon from 'sinon';
+
+describe('DatabaseSelect', () => {
+  it('should render', () => {
+    const wrapper = shallow(<DatabaseSelect />);
+    expect(
+      React.isValidElement(<DatabaseSelect />)
+    ).to.equal(true);
+  });
+
+  it('should call onChange', () => {
```

Comment:
could be more explicit with this test name...`it('calls onChange on select change')`